test/test_preprocessing.py

```diff
@@ -0,0 +1,13 @@
+import pandas as pd
+from src import preprocessing
+
+def test_read_file():
```

Comment:
This seems a bit like a pointless test?

Reply:
It wasn't supposed to be much more than a smoke test. The contents of the testing are not really the point, we just show that we can do CI with this stuff.